BUILD.md

```diff
@@ -24,3 +24,17 @@ tito tag
 ```
 tito build --rpm
 ```
+
+## Build openshift-ansible container image
```

Comment:
Oops, good catch.

I believe that moving the build instructions into `BUILD.md` makes sense, so I'd vote for updating the title/description in that file and link to it from the container image readme as you mention below.

If that's ok, I'll move the reference to `tito` into the section about building an RPM, and move "`RPM`" from the level 1 heading into the first level 2 heading